regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -151,15 +151,19 @@ var ChildView = Backbone.View.extend({
             var url = this.url,
                 hashPosition;
 
+            console.log(options);
+
             // if a hash has been passed in
             if (options && typeof options.scrollToId !== 'undefined') {
                 url += '#' + options.scrollToId;
-            }
-            else {
+            } else {
                 hashPosition = (typeof Backbone.history.fragment === 'undefined') ? -1 : Backbone.history.fragment.indexOf('#');
                 //  Be sure not to lose any hash info
                 if (hashPosition !== -1) {
-                    url += Backbone.history.fragment.substr(hashPosition);
+                    url = url.slice(0, hashPosition) + '#' + options.id;
```

Comment:
@cmc333333 very nice. To ease your worry a bit, the default is just `url` which already exists in memory. These ifs would only be if something needs appended to the url. 

This is actually likely to be closed as we haven't ever been appending that hash to the URL and I'm not sure that it solves the IE problem I was hoping it solves.